doc/walkthrough.md

```diff
@@ -0,0 +1,268 @@
+
+```
+gen=1
+equihashsolver=tromp
+mineraddress=YOUR_ADDRESS_HERE
+```
+
+Now, in order to detect those funds, we need to scan the chain:
+
+```bash
+λ cargo run --release --all-features -- wallet -w ../dev-wallet sync -s zecrocks
+```
+
+Having scanned the chain, we can now check the balance
+
+```bash
+λ cargo run --release --all-features -- wallet -w ../dev-wallet balance
+```
+
+You can explore more of the wallet commands available using:
+
+```bash
+λ cargo run --release --all-features -- wallet --help
+```
+
+PCZT Support
+------------
+
+It's possible to use `zcash-devtool` as a "simulated hardware wallet" via use
+of the `pczt` suite of commands. These commands provide reference
+implementations for PCZT creation, proving, signing, combining, inspection, and
+more.
+
+This section will simulate an interaction between an online wallet that scans
+the chain with a viewing key and an offline wallet that holds the signing keys.
+For the purpose of this simulation, the `../dev-wallet` wallet we've created
+will take on the role of our hardware device.
+
+First, we're going to set up our "online wallet" by exporting a viewing key
+from the development wallet, and using it to initialize a new view-only
+wallet.
+
+```bash
+λ cargo run --release --all-features -- wallet -w ../dev-wallet/ list-accounts
+```
+
+That produces output like the following:
+
+```bash
+Account ef4cde11-ac2b-4ac3-becf-3df518ee2c97
+     Name: ZDevTest
+     UIVK: <...>
+     UFVK: <dev_ufvk>
+     Source: Derived { derivation: Zip32Derivation { seed_fingerprint: SeedFingerprint(<dev_seed_fp>), account_index: AccountId(0) }, key_source: None }
+```
+
+Using the UFVK value provided there, we can initialize a new view-only wallet
+that we'll use as the online device in this exchange. After initializing it,
+we'll also sync that wallet. It is important to also include the seed
+fingerprint and HD account index, because we will need those to make it easy
+for the signing wallet to choose the correct key.
+
+```bash
+λ cargo run --release --all-features -- wallet -w ../view-wallet/ init-fvk \
+    --name ZDevView \
+    --fvk "<dev_ufvk>" \
+    --birthday 3274265 \
+    --seed-fingerprint "<dev_seed_fp>" \
+    --hd-account-index 0 \
+    -s zecrocks
+λ cargo run --release --all-features -- wallet -w ../view-wallet sync -s zecrocks
+```
+
+Asking for the balance here should show the same balance as we saw in our
+original wallet:
+
+```bash
+λ cargo run --release --all-features -- wallet -w ../view-wallet balance
+```
+
+We're now going to use the view-only wallet to create a PCZT. If you don't have
+a separate testnet wallet to receive these funds, you can always use the
+wallet's own address, or you can create a separate wallet to better simulate
+multiple independent wallets interacting.
+
+```bash
+λ cargo run --release --all-features -- pczt -w ../view-wallet create --address <...> --value 12340000 --memo "Hello from a PCZT!" > ../test_pczt.created
```

Comment:
for some reason when I copy paste this this is what I get

`cargo run --release --all-features -- pczt -w ../view-wallet create --address <...> --value 12340000 --memo "Hello from a PCZT! > ../test_pczt.created`


I think there's a faux quote mark on that text.